DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -112,13 +113,13 @@
               } else {
                 helpPage = "/help/Introduction.html";
               }
+              <li class="nhsuk-header__navigation-item">
+                <a class="nhsuk-header__navigation-link" href="@(Configuration["CurrentSystemBaseUrl"] + helpPage)" target="_blank">
+                  Help
+                  <partial name="_NhsChevronRight" />
+                </a>
+              </li>

```

Comment:
Off the top of my head, that feels like a lot of work just to go through adding a new section to most pages and double checking none have been missed and they all have the correct nav bar. On the other hand, if we are going to have a bunch more things like this that are relatively common but not omnipresent, then it would be better to sort this out now before we potentially add more pages. Do we think that there will be many more tabs that fall into a similar usage pattern as the Help tab?

Reply:
Oh, I meant more that the "Help" will be replaced with "Support" in most admin apps (Tracking system, Frameworks, probably Supervise etc), so it might make sense to dump Help vs Support into the application specific "NavigationMenuItems"?

Reply:
As discussed, not going to remove help atm.